src/applications/vaos/redux/selectors.js

```diff
@@ -66,3 +66,6 @@ export const selectFeatureVAOSServiceVAAppointments = state =>
 
 export const selectFeatureVAOSServiceCCAppointments = state =>
   toggleValues(state).vaOnlineSchedulingVAOSServiceCCAppointments;
+
+export const selectFeatureVariantTesting = state =>
+  toggleValues(state).vaOnlineSchedulingVariantTesting;
```

Comment:
I think this is from another PR?

Reply:
Oh good catch. I should have closed all my file tabs in VS Code before doing a checkout.